ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java

```diff
@@ -56,13 +57,16 @@ protected void onDraw(final Canvas canvas) {
     }
 
     private void drawBackgroundForButtons(final Canvas canvas) {
-        if(mPortrait) {
+        if(mShouldDrawBackgroundForButtons) {
```

Comment:
not sure if this should be inside the method. If I call the method with this name I expect it to draw and so it relies on something else. Better check before calling:
```Java
if(mShouldDrawBackgroundForButtons) {
    drawBackgroundForButtons(myCanvas);…
```